.github/workflows/assets.yaml

```diff
@@ -1,12 +1,59 @@
-name: Publish Docker image
+name: Publish binaries and Docker image
 
 on:
   release:
     types: [published]
 
 jobs:
-  # Push Iter8 image to Docker Hub
+  assets:
+    name: Publish binaries
+    runs-on: ubuntu-latest
+    steps:
+    - name: Install Go
+      uses: actions/setup-go@v4
+      with:
+        go-version: 1.19
+    - uses: actions/checkout@v3
+    - name: Build binaries
+      run: |
+        VERSION=${GITHUB_REF#refs/*/}
+        echo "Version: ${VERSION}"
+        make dist
```

Comment:
Given that only `iter8-linux-amd64.tar.gz` is needed by the Dockerfile, can we please change the `make dist` target in the Makefile to produce only this binary? This will save a bunch of time.